tests/Fixer/Whitespace/StatementIndentationFixerTest.php

```diff
@@ -1214,6 +1214,256 @@ public function foo() {
 echo 1;
 }',
         ];
+
+        yield 'multiline class constant' => [
+            '<?php
+class Foo
+{
+    const
+        FOO = 1;
+}',
+            '<?php
+class Foo
+{
+  const
+     FOO = 1;
+}',
+        ];
+
+        yield 'multiline class constant with visibility' => [
+            '<?php
+class Foo
+{
+    public const
+        FOO = 1;
+    protected const
+        BAR = 1;
+    private const
+        BAZ = 1;
+}',
+            '<?php
+class Foo
+{
+  public const
+     FOO = 1;
+  protected const
+         BAR = 1;
+  private const
+                BAZ = 1;
+}',
+        ];
+
+        yield 'multiline class constants' => [
```

Comment:
Since there's `multiline class constant` case, maybe it's better to name this one:

```suggestion
        yield 'multiline comma-separated class constants' => [
```

Now it clearly describes the intent.